mne/viz/_3d.py

```diff
@@ -1947,3 +1948,149 @@ def _dipole_changed(event, params):
                  params['idx'], params['dipole'], params['gridx'],
                  params['gridy'], params['ori'], params['coord_frame'],
                  params['zooms'], params['show_all'], params['scatter_points'])
+
+
+def plot_glass_brain(stc, subject, subjects_dir=None, src=None,
```

Comment:
I don't get why this code goes in _3d.py as it's not rendering anything in 3d.

you could put in a viz/source_estimate.py ?